messages/create.sandbox.md

```diff
@@ -123,3 +123,19 @@ The poll interval (%d seconds) can't be larger that wait (%d in seconds)
 # error.noCloneSource
 
 Could not find the clone sandbox name "%s" in the target org.
+
+# error.userGroupQueryFailed
+
+Could not query user group.
+
+# error.bothApexClassIdAndNameProvided
+
+Could not specify both apexClassId and apexClassName.
```

Comment:
```suggestion
You cannot specify both `apexClassId` and `apexClassName` in the definition file at the same time.
```

Reply:
```suggestion
You can't specify both `apexClassId` and `apexClassName` in the definition file at the same time.
```

Reply:
Love Cristian's suggestion, I just have a small tweak to it!